src/components/Board.js

```diff
@@ -16,14 +16,56 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    this.getCards()
+  }
+
+  getCards = () => {
+    axios.get('https://inspiration-board.herokuapp.com/boards/Naheed/cards')
+    .then((response) => {
+      this.setState({ cards: response.data });
+    })
+    .catch((error) => {
+      this.setState({ error: error.message });
+    });
+  }
+
+  addCard = (text, emoji) => {
+    axios.post(`https://inspiration-board.herokuapp.com/boards/Naheed/cards?text=${text}&emoji=${emoji}`)
+    .then((response) => {
+      this.getCards()
+    });
+  }
+
+  deleteCard = (id) => {
+    axios.delete(`https://inspiration-board.herokuapp.com/cards/${id}`, id)
+    .then((response) => {
+      this.getCards()
```

Comment:
like above ... instead of making another get request via `this.getCards`, we would prefer that you modify state

Reply:
I initially tried to update state, but it wouldn't re-render with the changes. When I did a hard refresh the new card would show/delete, but it wouldn't show up on its own after using setState. I wasn't able to find a solution so I went with another get request, but any ideas why setState wouldn't render?